Assets/Scripts/HealthController.cs

```diff
@@ -33,7 +34,28 @@ private void Start()
 
     public void DealDamage(DamageInfo info)
     {
+        if (!isNetworked || currentHealth <= 0)
```

Comment:
Just thought it wouldn't make sense to send messages about hits to objects with negative/no health

Reply:
I guess the practicality of this is that when you deal damage to something that is already dead, it won't necessarily be synchronized between clients. After thinking abit about it, I can't come to think of any situation where that would lead to a problem, so this is probably fine 👍

Reply:
It would lead to issues if we wanted to sync things like cacti and spent soda cans...
For exploding soda cans we might need to do this already, actually, since they die twice in order to explode 😔

Reply:
Removed this check for the reason state above. Soda cans should work now (though their position should be synced)